doc-ros2-documentation.yaml

```diff
@@ -0,0 +1,19 @@
+%YAML 1.1
+# ROS buildfarm ROS 2 documentation build file
+---
+documentation_type: docker_build
+doc_repositories:
+  ros2_doc:
+    url: https://github.com/ros2/ros2_documentation.git
+    branch: rolling
+jenkins_job_priority: 80
+jenkins_job_timeout: 30
+notifications:
+  emails:
+  - clalancette@openrobotics.org
+type: doc-build
+upload_credential_id: jenkins-agent
+upload_host: docs.ros.org
+upload_root: /var/www/docs.ros.org/en
```

Comment:
What is the intended url for the ROS 2 documentation root? I'm not sure that this upload root will deliver it.

Reply:
The target https://docs.ros.org/en/ .  Underneath that, there will be one directory per distribution, so we should end up with:

https://docs.ros/org/en/foxy
https://docs.ros.org/en/dashing
https://docs.ros.org/en/rolling

Do you happen to know where we would need to upload to accomplish that?